generators/vsi/templates/terraform/scripts/build_master.sh

```diff
@@ -20,3 +20,38 @@ mvn clean install
 mvn clean install
 target/liberty/wlp/bin/server package defaultServer --archive="{{deployment.name}}" --include=minify
 {{/has}}
+{{#has deployment.language 'GO'}}
+# get project name from manifest.yml
+export project_name="$(cat manifest.yml | grep -w "GOPACKAGENAME :" | sed 's/    GOPACKAGENAME : //')"
+# get the original folder name from the pwd command 
+original_folder_name="$(echo $(pwd) | sed 's/\/home\/pipeline\///')"
+# go into the directory above the folder w/ all the go files 
+cd /home/pipeline
+# zip the entire project  
+tar -czvf zippedProject.tar.gz $original_folder_name
+# create a project folder w/ the actual project name in GOPATH
+mkdir /go/src/$project_name
+# move the files there 
+mv -v zippedProject.tar.gz /go/src
+# go to that directory in GOPATH
+cd /go/src/
+# rename the folder to the project name 
+tar -xzvf zippedProject.tar.gz
+mv $original_folder_name $project_name
+# go into that folder and move everything up one directory 
+cd $project_name
+mv  -v $original_folder_name/* /go/src/$project_name/ 
+# get/run dep 
+go get -u github.com/golang/dep/cmd/dep
```

Comment:
One thing you can do to shorten this is to combine a few lines that are doing similar things. Such as for lines 45-47:

`go get -u github.com/golang/dep/cmd/dep && dep init && dep ensure`